roles/edpm_nova/molecule/default/prepare.yml

```diff
@@ -65,11 +65,19 @@
               proto: tcp
               dport: 22
     - name: install libvirt
-      import_role:
+      ansible.builtin.import_role:
         name: osp.edpm.edpm_libvirt
     - name: ensure /etc/multipath.conf exists
       become: true
       ansible.builtin.file:
         path: /etc/multipath.conf
         state: touch
         mode: "0644"
+    - name: Create Ceph client file directory
+      become: true
+      ansible.builtin.file:
+        path: '/var/lib/openstack/config/ceph/'
+        state: directory
+        owner: root
+        group: root
+        mode: 0755
```

Comment:
quote the octal number[1]

[1] https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#parameter-mode

Reply:
this is a cherry-pick of an already merged patch we should not modify it

Reply:
Sorry Sean, i missed that it wasn't for `main` branch.